sycl/include/sycl/handler.hpp

```diff
@@ -2532,6 +2540,8 @@ class __SYCL_EXPORT handler {
 private:
   std::shared_ptr<detail::handler_impl> MImpl;
   std::shared_ptr<detail::queue_impl> MQueue;
+  std::shared_ptr<ext::oneapi::experimental::detail::graph_impl> MGraph;
```

Comment:
Realized i'm not actually using this new member variable anywhere (I assumed i'd need it for `hander::finalize`, but that code path was only called called in record & replay mode where we get the graph from a queue regardless.)

I'll try remove this on Monday, which would mean the new handler can be `handler::handler()`

Reply:
indeed the graph member can be removed, updated